pageserver/src/http/openapi_spec.yml

```diff
@@ -351,6 +351,13 @@ paths:
         schema:
           type: string
           format: hex
+      - name: allow_ignored
```

Comment:
I've used `detach_ignored` as a parameter name in the `mgr` code, should we use the same name on HTTP level too?